pom.xml

```diff
@@ -141,7 +148,7 @@
 				<configuration>
 					<serverId>ossrh</serverId>
 					<nexusUrl>https://s01.oss.sonatype.org/</nexusUrl>
-					<autoReleaseAfterClose>true</autoReleaseAfterClose>
+					<autoReleaseAfterClose>false</autoReleaseAfterClose>
```

Comment:
This thing will not be published on MavenCentral, are you sure we need this plugin?

Reply:
Deleted dependency